src/rgw/rgw_op.cc

```diff
@@ -6159,6 +6166,8 @@ void RGWInitMultipart::execute(optional_yield y)
     upload_id = upload->get_upload_id();
   }
   s->trace->SetAttribute(tracing::rgw::UPLOAD_ID, upload_id);
+  multipart_trace->UpdateName(tracing::rgw::MULTIPART + upload_id);
```

Comment:
why do you serialize before the name update?

Reply:
the span name isn't part of the span context.

Reply:
ok. but we can still put that in one place?